### DISCUSSION TITLE
Re-add support for "fail-background.png" from custom skins

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#25508] [Ideas] [answered]
In stable, skins could add a custom graphic called "fail-background" that would fade-in behind the game over menu. In lazer, this skinning element is not used, instead showing a dimmed view of the map. I am aware a lot of skinning support is still WIP but as such features are developed I'd like to show my support for ensuring that this particular element is eventually supported (even if other aspects of the fail screen aren't, such as the custom buttons). As a side note, I may be wrong, but I could've sworn a much earlier build did support this skin element (before the current fail animation with the zoom-out was added).

See the attached video, which shows the game over state in both stable and lazer using the same skin (ebo-mini):

https://github.com/ppy/osu/assets/76608775/a60352b0-1154-486c-b79e-723d76b53b2f

**Comment by peppy** [comment, marked as answer]
Not just the background, but the buttons and the whole fail screen needs skinning support. This will come later.
